galasa-extensions-parent/dev.galasa.ras.couchdb/src/main/java/dev/galasa/ras/couchdb/internal/CouchdbValidatorImpl.java

```diff
@@ -298,14 +300,14 @@ private boolean checkViewString(JsonObject view, String field, String value) {
     }
 
     private void checkVersion(String version, int minVersion, int minRelease, int minModification)
```

Comment:
checkVersionAboveMinimumVersion ?

Why isn't a version a class, rather than 3 primitive type ints ?

Then we could just compare the version against a minimum one using a comparator like '>'